README.md

```diff
@@ -1484,3 +1485,30 @@ webServer.onNotFound([](AsyncWebServerRequest *request) {
   }
 });
 ```
+
+### Path variable
+
+With path variable you can create a custom regex rule for a specific parameter in a route. 
+For example we want a `sensorId` parameter in a route rule to match only a integer.
+
+```cpp
+  server.on("^\\/sensor\\/([0-9]+)$", HTTP_GET, [] (AsyncWebServerRequest *request) {
+      String sensorId = request->pathArg(0);
+  });
+```
+*NOTE*: All regex patterns starts with `^` and ends with `$`
+
+To enable the `Path variable` support, you have to define the buildflag `-DASYNCWEBSERVER_REGEX`.
+
+For Arduino IDE modify `platform.local.txt`:
```

Comment:
where is that platform.local.txt?

Reply:
I found out that platform.local.txt is not needed.

For ardruino IDE:
```cpp
#define ASYNC_WEBSERVER_REGEX
#include <ESPAsyncWebServer.h>
```

Reply:
I will switch back to the platform.local.txt method. 
I do not prefer that method, but it is the only method that works with the Arduino IDE

Reply:
yes, unfortunately we can not add menus from the libraries (or something like that). Arduino is quite more basic that most of the editors that I use. Users know to poke around. That is how ESP8266 has SSL support in ESPAsyncTCP